actioncable/app/assets/javascripts/action_cable.coffee.erb

```diff
@@ -33,3 +33,12 @@
     if @debugging
       messages.push(Date.now())
       console.log("[ActionCable]", messages...)
+
+# Since we are `module.exports`-ing below, in order to make the internal `ActionCable class`
+# recognize the `ActionCable` constant, we need to place the constant on the `window`, so that
+# way it will be recognized.
+window.ActionCable = @ActionCable
+
+# We need to module.exports here, in order to make ActionCable a "true Node.js module".
+# This makes it so that way an `Import` or `require` will load in the `ActionCable` module.
+module?.exports = @ActionCable
```

Comment:
Jon actually did this as a result of our ongoing conversation about how to expose the module to NPM -- he was just concatenating the JS files into dist but that didn't work because the `ActionCable` var was undefined and as a result trying to use the NPM package would just auto-error. It's worth having a discussion about which module loaders to support, but IMO this is essential to the PR even doing anything useful.